tests/boot/with_mcumgr/pytest/test_security.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import pytest
+import logging
+
+from pathlib import Path
+from twister_harness import Device, MCUmgr
+from conftest import (
+    wait_for_message,
+    west_sign_with_imgtool,
+    find_in_config,
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    'key_file',
+    [
+        '',
+        'root-ec-p256.pem',
```

Comment:
pretty sure this won't work with mcuboot in zephyr at present due to ARM breaking imgtool, will submit a PR later today updating the mcuboot revision to include a fix

Reply:
But this `key_file` is used only in `west sign` fixture:
https://github.com/zephyrproject-rtos/zephyr/blob/95e1df0d430ac90e3fa148dd67860a80df0bffa1/tests/boot/with_mcumgr/pytest/conftest.py#L30
and is not passed to west command if is empty.
West sign command is then called without `--key`, so the image will not be signed.